# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/gobject-introspection/hotfix-distutils.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- gobject-introspection-1.78.1/giscanner/utils.py.vanilla	2024-02-12 17:23:07.052000000 +0000
+++ gobject-introspection-1.78.1/giscanner/utils.py	2024-02-12 17:23:26.478000000 +0000
@@ -26,6 +26,7 @@
 import sys
 import time
 import giscanner.pkgconfig
+import setuptools
 from typing import Dict
 
 
